src/containers/profile/profile.js

```diff
@@ -20,29 +16,36 @@ export const Profile = () => {
     role,
     close_button,
     menu,
-    hidden_text
+    hidden_text,
+    button,
+    active_button,
   } = styles;
 
+  const currentTab = window.location.pathname;
   
-  const toggleButton = (elem) => {
-    setStatus(elem);
-    setComponent(elem);
-
-  }
+  const tabs = {
+    public: '/profile-public',
+    courses: '/profile-courses',
+    account: '/profile-account',
+    settings: '/profile-settings',
+  };
 
   const toggleMenu = () => {
-    if(menus.marginLeft === '0'){
-      setMenu({ marginLeft: '-105%' })
-    }else{
-      setMenu({ marginLeft: '0' })
+    if (menus.marginLeft === '0') {
+      setMenu({ marginLeft: '-105%' });
```

Comment:
Why are you not using `classList.toggle('className')`?

Reply:
done